docs/blog/2018-2-16-how-to-build-a-website-with-react/index.md

```diff
@@ -29,28 +26,29 @@ React is not picky; it is happy to be used in only parts of your site, so you ca
 
 ## Component-Based
 
-React components and subcomponents tend to come from breaking your website down into the smallest bits possible, using the [single responsibility principle](https://en.wikipedia.org/wiki/Single_responsibility_principle). 
+React components and subcomponents tend to come from breaking your website down into the smallest bits possible, using the [single responsibility principle](https://en.wikipedia.org/wiki/Single_responsibility_principle).
+
+For example, in a To-Do list, the hierarchy of components would include:
 
-For example, in a To-Do list, the hierarchy of components would include: 
 * Whole list
-    * Title
-    * add a to-do line
-    * to-do line
-        * subtasks within to-dos
-    * show completed to-dos button
+  * Title
+  * add a to-do line
+  * to-do line
+    * subtasks within to-dos
+  * show completed to-dos button
```

Comment:
Hey @KyleAMathews and @m-allanson, sorry for not merging this myself, got sidetracked a little. :-/

What I wanted to check before merging is what I just checked now: I have Prettier(-eslint) set up to run everytime I save, thus all the reformatting around here w/o explicitly running `format-markdown`. Before merging I noticed that the list above doesn't display correctly after Prettier changed indention from four to two spaces (after/before with four spaces): 

![image](https://user-images.githubusercontent.com/21834/36334944-ea6ae532-137c-11e8-8c01-a654a01357b7.png)

![image](https://user-images.githubusercontent.com/21834/36335063-985ce096-137d-11e8-82ce-e6cd0b3dab13.png)

Will take a look at `using-remark` and open an issue.